config/site/support/llm_content_extractor.rb

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 Block, Inc.
+#
+# Use of this source code is governed by an MIT-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+#
+# frozen_string_literal: true
+
+require "nokogiri"
```

Comment:
```suggestion
```

Perhaps a moot point if you merge extractors as I suggest in another comment, but `Nokogiri` isn't referenced below so no need to require it.